ui/pages/confirmations/hooks/alerts/useBlockaidAlerts.test.ts

```diff
@@ -71,11 +55,16 @@ describe('useBlockaidAlerts', () => {
   });
 
   it('returns alerts when there is a valid PersonalSign confirmation with a security alert response', () => {
-    const mockCurrentState = getMockCurrentState({
-      signatureSecurityAlertResponses: {
-        'test-id-mock': mockSecurityAlertResponse,
+    const mockCurrentState = getMockPersonalSignConfirmStateForRequest(
+      currentConfirmationMock as unknown as SignatureRequestType,
```

Comment:
```suggestion
      currentConfirmationMock,
```

Similarly to the [comment above](https://github.com/MetaMask/metamask-extension/pull/26649#discussion_r1739633595), we can use TransactionType in the currentConfirmationMock.type to apply the suggestion

Reply:
Hey @digiwand :

TransactionType does not seems to work either:

<img width="943" alt="Screenshot 2024-09-03 at 7 27 13 AM" src="https://github.com/user-attachments/assets/3b4a453b-3f2b-4c06-a69a-f11d23bb43f1">

Reply:
Thanks @digiwand : I see ur point, I fixed it in the PR.